client/src/components/FriendsDashboard/User.js

```diff
@@ -58,12 +60,26 @@ export default function Suggestions ({ user }) {
             console.log(err.response.data.msg);
         }
     };
+
+    const goToProfile = () => {
+        setSelectedUser(user);
+        
+        history.push({
+            pathname: `/${user._id}/profile`,
```

Comment:
this makes sense to identify the user_id in the route but similarly, would you lose the `user` object here if you refresh your page? 

if you do, i think the next best option is to do an API call to the backend to retrieve user info based off of the id you pass to it

Reply:
Thanks Bonnie, let me know if the code below is better!